src/main/scala/scalikejdbc/bigquery/QueryExecutor.scala

```diff
@@ -1,32 +1,23 @@
 package scalikejdbc.bigquery
 
-import com.google.cloud.bigquery.BigQuery.{QueryOption, QueryResultsOption}
-import com.google.cloud.bigquery.{BigQuery, QueryResponse}
+import com.google.cloud.bigquery.{BigQuery, TableResult}
 import scalikejdbc._
 
-import scala.concurrent.duration._
-
 class WrappedQueryResponse(
-  private[bigquery] val underlying: QueryResponse,
+  private[bigquery] val underlying: TableResult,
   private[bigquery] val rsTraversable: Traversable[WrappedResultSet])
 
 class QueryExecutor(bigQuery: BigQuery, config: QueryConfig) {
 
   def execute(statement: SQLSyntax): WrappedQueryResponse = {
 
-    val builder = QueryRequestBuilder(statement)
+    val request = QueryRequestBuilder(statement)
       .setUseLegacySql(config.useLegacySql)
       .setUseQueryCache(config.useQueryCache)
+      .build()
 
-    val queryOptions = Seq(
-      QueryOption.of(QueryResultsOption.maxWaitTime(30.minutes.toMillis)) // TODO: make configurable
-    )
-
-    val request = builder.build()
-
-    val response = bigQuery.query(request, queryOptions: _*)
-
-    val rs = new BqResultSet(response.getResult)
+    val response = bigQuery.query(request)
+    val rs = new BqResultSet(response)
```

Comment:
Is there any reason to remove maxWaitTime option ?
I think it is better to reserve this option.

```scala
package scalikejdbc.bigquery

import com.google.cloud.bigquery.BigQuery.QueryResultsOption
import com.google.cloud.bigquery.{BigQuery, JobInfo, TableResult}
import scalikejdbc._

import scala.concurrent.duration._

class WrappedQueryResponse(
  private[bigquery] val underlying: TableResult,
  private[bigquery] val rsTraversable: Traversable[WrappedResultSet])

class QueryExecutor(bigQuery: BigQuery, config: QueryConfig) {

  def execute(statement: SQLSyntax): WrappedQueryResponse = {

    val request = QueryRequestBuilder(statement)
      .setUseLegacySql(config.useLegacySql)
      .setUseQueryCache(config.useQueryCache)
      .build()
    
    val queryResultsOptions = Seq(
      QueryResultsOption.maxWaitTime(30.minutes.toMillis) // TODO: make configurable
    )
    
    val job = bigQuery.create(JobInfo.of(request))

    val response = job.getQueryResults(queryResultsOptions: _*)
    val rs = new BqResultSet(response)

    new WrappedQueryResponse(response, new ResultSetTraversable(rs))
  }
}
```

Reply:
I think `Job#getQueryResults` wait for job completed irrespective of `maxWaitTime`

`maxWaitTime` used [HttpBigQueryRpc#GetQueryResultsResponse](https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/spi/v2/HttpBigQueryRpc.java#L398) , but `Job#getQueryResults` process polling in [this Block](https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQueryImpl.java#L682) for job completion.

So, I thought it was unnecessary and removed.

discussion: https://github.com/GoogleCloudPlatform/google-cloud-java/issues/2543